modules/mediawiki/files/bin/deploy-mediawiki.py

```diff
@@ -226,8 +231,17 @@ def run(args: argparse.Namespace, start: float) -> None:
             exitcodes.append(run_command(cmd))
         non_zero_code(exitcodes)
         if args.l10n:  # setup l10n
+            if args.lang:
+                for language in str(args.lang).split(','):
+                    if not tag_is_valid(language):
+                        raise ValueError(f'{language} is not a valid language.')
+
+                lang = f'--lang={args.lang}'
+            else:
+                lang = ''
+
             postinstall.append(f'sudo -u www-data php /srv/mediawiki/w/maintenance/mergeMessageFileList.php --quiet --wiki={envinfo["wikidbname"]} --output /srv/mediawiki/config/ExtensionMessageFiles.php')
-            rebuild.append(f'sudo -u www-data php /srv/mediawiki/w/maintenance/rebuildLocalisationCache.php --quiet --wiki={envinfo["wikidbname"]}')
+            rebuild.append(f'sudo -u www-data php /srv/mediawiki/w/maintenance/rebuildLocalisationCache.php {lang} --quiet --wiki={envinfo["wikidbname"]}')
```

Comment:
We should probably split this off into a generate method. Should I do a pull request for that?